src/main/java/core/mate/academy/service/producers/BulldozerProducer.java

```diff
@@ -0,0 +1,25 @@
+package core.mate.academy.service.producers;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private static final int NUM_OF_MACHINES = 3;
+    private static final int MAX_POWER = 300;
+    private static final int MIN_POWER = 100;
+    private final Random random = new Random();
+
+    @Override
+    public List<Machine> get() {
+        List<Machine> machines = new ArrayList<>();
+        int randomPower = random.nextInt(MIN_POWER, MAX_POWER);
+        for (int i = 0; i < NUM_OF_MACHINES; i++) {
+            machines.add(new Bulldozer(randomPower));
+        }
```

Comment:
BTW, power is still the same for all bulldozers you produce here, for it to change you need to call `nextInt()` inside the loop, to get a new number every time, because if it is outside of loop - int var `randomPower` is not going to change magically by itself
```suggestion
        for (int i = 0; i < NUM_OF_MACHINES; i++) {
            int randomPower = random.nextInt(MIN_POWER, MAX_POWER);
            machines.add(new Bulldozer(randomPower));
        }
```